frontend/e2e-test/test/e2e/specs/0_citizen/citizen-daycare-application-create.spec.ts

```diff
@@ -26,17 +27,35 @@ import {
   Fixture
 } from '../../dev-api/fixtures'
 import {
+  FormInput,
   fullDaycareForm,
   minimalDaycareForm
 } from '../../utils/application-forms'
 import { add, sub } from 'date-fns'
 import CitizenDecisionsPage from '../../pages/citizen/citizen-decisions'
 import CitizenDecisionResponsePage from '../../pages/citizen/citizen-decision-response'
+import path from 'path'
+import { RequestLogger } from 'testcafe'
+
+const logger = RequestLogger(
+  { url: /\/attachments\/[a-fA-F0-9-]+\/download$/, method: 'GET' },
+  { logResponseHeaders: true }
+)
+
+const testFilePath = '../../assets/test_file.jpg'
+const testFileName = path.basename(testFilePath)
+const testFileDispositionRegex = new RegExp(
+  `attachment;\\s*filename=${testFileName.replace(
+    /[|\\{}()[\]^$+*?.]/g,
+    '\\$&' // Escape characters in filename to make a valid regular expression
+  )}`
+)
```

Comment:
Instead of this pretty complicated regex would a simple string comparison or a `String.includes` be enough?

Reply:
Yeah, sounds reasonable -- I have a bit of a blind spot for complicated regexes.